src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/SerializeAbsoluteContext.cs

```diff
@@ -7,7 +7,7 @@ namespace System.ComponentModel.Design.Serialization
     /// <summary>
     ///  The ComponentSerializationService supports "absolute" serialization,
     ///  where instead of just serializing values that differ from an object's
-    // default values, all values are serialized in such a way as to be able
+    /// default values, all values are serialized in such a way as to be able
```

Comment:
The rest of the lines have the wrong indentation, but this one should at least match.

```suggestion
    ///  default values, all values are serialized in such a way as to be able
```